Functions/PesterState.ps1

```diff
@@ -6,7 +6,7 @@ function New-PesterState
         [String[]]$TestNameFilter,
         [System.Management.Automation.SessionState]$SessionState,
         [Switch]$Strict,
-        [Switch]$Quiet,
+        [Pester.OutputTypes]$Show = 'All',
```

Comment:
@iSazonov This is for compatibility reasons. Changing this from `All` to something else would be a breaking change for users.